03/app.js

```diff
@@ -1,3 +1,13 @@
 export default function randomNumber(min, max) {
-
-}
\ No newline at end of file
+    if (min === max) {
+        return min;
+    }
+    if (isNaN(min) || isNaN(max)) {
+        throw new Error('Property have to be number');
+    }
+    if (min > max) {
+        throw new Error('Wrong range');
+    }
+   
+    return Math.floor(Math.random() * (max - min) + max)
```

Comment:
Testy wyłapały u mnie błąd ;P Nie wiem dlaczego u Ciebie to nie wyszło :P
Zamiast `+ max` powinno być `+ min` :D